lib/index.ts

```diff
@@ -20,6 +20,20 @@ import * as tar from 'tar-stream';
 import { Readable, Writable } from 'stream';
 
 const noop = () => {};
+const pathSepRE = new RegExp(escapeRE(path.sep), 'g');
+const normalize: (p: string) => string =
+	path.sep === '/' ? path.normalize : normalizeNonPosix;
+
+/**
+ * Alternative path.normalize() function that uses '/' as a path separator in
+ * non-POSIX systems like Windows (which uses a backslash). This is useful for
+ * tar files/streams as the path separator should always be '/', regardless of
+ * platform/OS, as per tar spec:
+ * https://www.gnu.org/software/tar/manual/html_node/Standard.html
+ */
+function normalizeNonPosix(p: string): string {
+	return path.normalize(p).replace(pathSepRE, '/');
+}
```

Comment:
You can use `path.posix.normalize` here.

Reply:
What?!? Where did that come from?! I had just saved several lines of code using lodash following @hedss comment, now I can delete a couple more functions too. What next? I guess if I just wait another half an hour, someone else will add a comment: _"oh, you're using `tar-utils`? you don't need that, just `import genie` and you're done!"_ or maybe: _"oh, all of that to fix the CLI under Windows? you don't need the CLI, just say 'Siri, push the app to balena' and it's all sorted"._
LOL :-)

Reply:
Unfortunately, we're not using python https://xkcd.com/353/ :)